src/python/WMCore/BossAir/Plugins/SimpleCondorPlugin.py

```diff
@@ -314,6 +314,10 @@ def complete(self, jobs):
                                     if errMsg:
                                         logOutput += "\n\n%s\n" % errMsg
 
+                                    errMsg = matchObj.group('ERROR')
```

Comment:
Isn't it going to fetch part of the WMException message?

Reply:
I change the reg expression.
https://github.com/dmwm/WMCore/pull/8313/files#diff-0ed75c44d8f6712727213d921d2f53e4R50

So it will look for WMExeption or ERROR. 
This ERROR seems to be not part of WMException it fails before it create WMException (mostly syntax error)